### PR TITLE
Streamline cep work flow

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: text/000-coq-enhancement-proposal.md [hunk 1]
```diff
@@ -14,33 +12,45 @@ tracker hardly converge on a clear picture.  CEP are meant to produce clear
...
 users, implemented and finally used as a reference.
 
-A CEP is a document like this one.  Useful fields in the header are Title,
-Drivers (people interested in the proposal and working on it) and Status.
-Status shall be one of Draft, Candidate, Accepted, Rejected, Obsolete.
+A CEP is a document like this one.  
+Useful sections are
+- title: a catchy title is key to success ;-)
+- driver: your name/contact
+- summary: from reading this part one should understand if he is interested and/or competent on the subject of the CEP
```
Comment (Zimmi48) [inline text/000-coq-enhancement-proposal.md]: Same remark here of course.

**Comment by gares** [comment]
I'd prefer a text field, rather than moving the file, to signal that a CEP in actually implemented.
One could even remove implemented CEP, and put them into the manual or in dev/doc.
Some CEPs, i.e. CEP3, don't really need an implementation.

**Comment by Zimmi48** [comment]
Isn't the conclusion of the working group that this PR should be merged? (And also that PRs should be created for non-accepted CEPs.)
I would also like to suggest adding a link from the README of this repo, to this particular (meta-)CEP.

**Comment by gares** [comment]
Sure, but I'm on VAC so I will not be able to do it before next week.  If you have time and will, go ahead.

**Comment by Zimmi48** [comment]
OK, but *I* don't have any write rights. But anyways, there is no hurry.

**Comment by Zimmi48** [comment]
@gares After all, can you click the merge button now so that I can fork the repo and follow the new process? And postpone the rest (moving old CEPs to PRs) and updating the README and marking implemented CEPs as such, later when you have time...

**Pull request closed without merging**